clang/test/dpct/helper_files_ref/include/rng_utils.hpp

```diff
@@ -174,6 +176,235 @@ template <typename engine_t> class rng_generator {
 };
 
 } // namespace device
+
+namespace host {
+enum class random_engine_type {
+  philox4x32x10,
+  mrg32k3a,
+  mt2203,
+  mt19937,
+  sobol
+};
+
+/// The random number generator on host.
+class rng_host {
```

Comment:
For consistency with previous api already introduced:
```suggestion
class rng_generator {
```

Reply:
Fixed.